src/content/structured/components/side-nav/code.mdx

```diff
@@ -747,6 +749,115 @@ export const withReactRouter = [
   </MemoryRouter>
 </ComponentPreview>
 
+### With MUI icons in SlottedSVG
```

Comment:
```suggestion
### With MDI icons in SlottedSVG
```
MUI icons are different to Material Design Icons (MDI). I also think it would be good to expand the acronym before we abbreviate